lib/generator.js

```diff
@@ -63,8 +65,9 @@ class Generator {
    * @param {String[]} [options.noOverwriteGlobs] List of globs to skip when regenerating the template.
    * @param {String[]} [options.disabledHooks] List of hooks to disable.
    * @param {String} [options.output='fs'] Type of output. Can be either 'fs' (default) or 'string'. Only available when entrypoint is set.
+   * @param {String} [options.forceWrite] Force writing of the generated files to given directory even if it is a git repo with unstaged files or not empty dir.
```

Comment:
Should this parameter have a default value of `false`?

Reply:
actually I removed it in my last commit because if not specified, it will anyway be `undefined` and will still work as expected. Condition will not pass until `true` anyway

Reply:
Yes, but then [this documentation](https://github.com/asyncapi/generator/pull/223/files#diff-88333ba11114e014e87198d04680144dR33) will not render that the default value is `false`. Just because `undefined` casts to `false` when checking on conditions it doesn't mean `undefined` is correct, right? Think about it as if we had types. Some day we'll have them 😉

Reply:
![](https://media.giphy.com/media/NQRuMoHOLYX6rgD2nD/giphy.gif)

I like `undefined`, don't treat it like this, it is enough that all those Java devs are mocking it 😄 
I will revert to get default back in 😄

Reply:
@fmvilas default added, check out my last commit

Reply:
LOL, I like it too. I just don't think this is its right place. Thanks man. Resolving.